tests/extensions/test_desinventar.py

```diff
@@ -0,0 +1,81 @@
+import json
+from os import makedirs
+from typing import List, Tuple, TypedDict
+from unittest import TestCase
+
+from parameterized import parameterized
+
+from pystac_monty.sources.desinventar import (
+    DesinventarDataSource,
+    DesinventarTransformer,
+)
+from tests.conftest import get_data_file
+
+
+class DesinventarData(TypedDict):
+    zip_file_url: str
+    country_code: str
+    iso3: str
+
+
+class DesinventarScenario(TypedDict):
+    name: str
+    data: DesinventarData
+
+
+nepal_data: DesinventarScenario = {
+    "name": "Nepal data",
+    "data": {
+        "zip_file_url": "https://www.desinventar.net/DesInventar/download/DI_export_npl.zip",
+        "country_code": "npl",
+        "iso3": "npl",
+    },
+}
+
+chile_data: DesinventarScenario = {
+    "name": "Chile",
+    "data": {
+        "zip_file_url": "https://www.desinventar.net/DesInventar/download/DI_export_chl.zip",
+        "country_code": "chl",
+        "iso3": "chl",
+    },
+}
+
+
+def load_scenarios(
+    scenarios: list[DesinventarScenario],
+) -> List[Tuple[str, DesinventarTransformer]]:
+    transformers: List[Tuple[str, DesinventarTransformer]] = []
+
+    for scenario in scenarios:
+        data = scenario["data"]
+        data_source = DesinventarDataSource.from_url(data["zip_file_url"], data["country_code"], data["iso3"])
+
+        transformer = DesinventarTransformer(data_source)
+        transformers.append((data["country_code"], transformer))
+
+    return transformers
```

Comment:
```suggestion
grenada_data: DesinventarScenario = {
    "name": "Grenada subset",
    "data": {
        "zip_file_url": "https://github.com/IFRCGo/monty-stac-extension/raw/refs/heads/desinventar/model/sources/Desinventar/DI_export_grd_subset.zip",
        "country_code": "grd",
        "iso3": "grd",
    },
}


def load_scenarios(
    scenarios: list[DesinventarScenario],
) -> List[Tuple[str, DesinventarTransformer]]:
    transformers: List[Tuple[str, DesinventarTransformer]] = []
    for scenario in scenarios:
        data = scenario["data"]
        # download zip file in temp folder
        response = requests.get(data["zip_file_url"])
        tmp_zip_file = tempfile.NamedTemporaryFile(suffix=".zip", delete=False)
        tmp_zip_file.write(response.content)
        data_source = DesinventarDataSource(tmp_zip_file, data["country_code"], data["iso3"])

        transformers.append((data["country_code"], DesinventarTransformer(data_source)))
    return transformers
```

Test are too long and not recorded so I propose to have subsets to make more relevant tests